src/pages/MainPage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import './mainpage.css'
+import DrawerBar from '../components/DrawerBar'
+
+const MainPage = () => {
+    
+
+    return (
+        <>
+            <Box className='container' sx={{
+                marginTop: 15,
+            }}>
+                <Paper
+                    elevation={8}
+                    sx={{
+                        width: 1200,
+                        height: 600,
```

Comment:
There are soo many fixed sizes.. Is there a reason why you are doing that and not having a more flexible width approach?